pull_request_template.md

```diff
@@ -38,8 +44,11 @@
 # Checklist
 
 - [ ] The implementation has been tested locally
+- [ ] The author of this PR is the Assignee
+- [ ] The implementation has been tested in staging (and the person who is doing it is an assignee)
+  - [ ] This cannot be tested on staging, and the reasons why are in the `How to test / reproduce` section
```

Comment:
Is this extra space intended?

Reply:
Yes that is deliberate. This is a sub section of the above point, so indenting it to make that obvious